projects/adaq8092_fmc/Readme.md

```diff
@@ -0,0 +1,8 @@
+# ADAQ8092_FMC HDL Project
+
+Here are some pointers to help you:
+  * [Board Product Page](https://www.analog.com/adaq8092)
+  * Parts : [2 Channels, 14-Bit, 105 MSPS, Analog-to-Digital Converter](https://www.analog.com/adaq8092)
+  * Project Doc: https://wiki.analog.com/resources/eval/user-guides/adaq8092/quickstart/zedboard
```

Comment:
This should probably be pointing to Project Doc: https://wiki.analog.com/resources/eval/user-guides/adaq8092